pkg/storage/util_test.go

```diff
@@ -174,7 +228,11 @@ func (m mockChunkStoreClient) PutChunks(ctx context.Context, chunks []chunk.Chun
 func (m mockChunkStoreClient) GetChunks(ctx context.Context, chunks []chunk.Chunk) ([]chunk.Chunk, error) {
 	var res []chunk.Chunk
 	for _, c := range chunks {
-		for _, sc := range m.chunks {
+		for index, sc := range m.chunks {
+			// We have decalared 7 chunks in the mock store. Mark 5th chunk as invalid chunk
+			if len(chunks) == 1 && index == 5 {
+				return nil, chunkenc.ErrInvalidChecksum
+			}
```

Comment:
I know this is not clean coding. However, this was the best and easiest I could think of. I am looking for other better ways to say `invalid chunk` from the loop.

Reply:
I thinik it's fine.

Reply:
Actually do you need this ? not sure it is executed ? I don't think `GetChunks` is used ?

Reply:
Cortex has this as an interface https://github.com/cortexproject/cortex/blob/master/pkg/chunk/chunk_store_utils.go#L156
Since we are passing mockStorageClient, it will call GetChunks here declared in util_test

Reply:
Yep you're right !